main_spec.tex

```diff
@@ -128,13 +128,25 @@ \subsection{Communication Management Routines}
 additionally provide thread isolation, eliminating overheads resulting from
 thread interference.
 
-Context handles are of type \CTYPE{shmem\_ctx\_t} and are valid for
-language-level assignment and equality comparison.  A handle to the desired context is
-passed as an argument in the \Cstd \CTYPE{shmem\_ctx\_*} and type-generic API
-routines.  API routines that do not accept a context argument operate on the
+\oldtext{A handle to the desired context}
+\newtext{A communication context is specified by a context handle, which} is
+passed as an argument in the \Cstd \CTYPE{shmem\_ctx\_*} and type-generic \ac{API}
+routines.  \ac{API} routines that do not accept a context \newtext{handle} argument operate on the
 default context.  The default context can be used explicitly through the
 \LibHandleRef{SHMEM\_CTX\_DEFAULT} handle.
 
+Context handles are of type \CTYPE{shmem\_ctx\_t} and \newtext{may be used} \oldtext{are valid} for
+language-level assignment and equality comparison.
+\newtext{
+A context handle may be initialized to, assigned, or equality-compared with
+the value \CONST{SHMEM\_CTX\_INVALID} to indicate or test whether the handle
+represents an invalid context.
```

Comment:
Same concern with respect to "equalith-compared with ... to test whether the handle represents *an* invalid context."